Fix build CI

CI is failing with: 
```
Logging into your Netlify account...
Opening https://app.netlify.com/authorize?response_type=ticket&ticket=16etsv
- Waiting for authorization...
 ›   Error: Timed out waiting for authorization. If you do not have a Netlify account, please create one at https://app.netlify.com/signup, then run netlify login again.
Error: Process completed with exit code 1.
```

--- a/.github/workflows/netlify.yml
+++ b/.github/workflows/netlify.yml
@@ -16,7 +16,7 @@ jobs:
         run: |
           npm install
           npm run build
-          npx netlify deploy --dir=build --site=${{ secrets.SITE_ID }} --auth=${{ secrets.NETLIFY_AUTH_TOKEN }}
+          npx netlify deploy --dir=build --site=${{ secrets.SITE_ID }} --auth="${{ secrets.NETLIFY_AUTH_TOKEN }}"
         env:
           CI: true
           NODE_ENV: production

